### PR TITLE
Fix unhandled exception on iOS when exitPointerLock is called.

### PR DESCRIPTION
**Pull request by tcobbs-bentley**
I don't know why document.exitPointerLock is undefined when running on iOS, but this should prevent it from being called if it is undefined.

### CODE REVIEW

File: core/frontend/src/tools/ViewTool.ts [hunk 1]
```diff
@@ -2128,7 +2128,7 @@ class ViewLookAndMove extends ViewNavigate {
         if (pixel.distanceFraction < 0)
           continue; // No geometry at location...
 
-        const hitPointWorld = vp.getPixelDataWorldPoint({pixels, x: testPoint.x, y: testPoint.y, preserveModelDisplayTransforms: true});
+        const hitPointWorld = vp.getPixelDataWorldPoint({ pixels, x: testPoint.x, y: testPoint.y, preserveModelDisplayTransforms: true });
```
Comment (tcobbs-bentley) [inline core/frontend/src/tools/ViewTool.ts:2131]: VS Code made this change automatically. I'm not sure if that is due to my global settings or due to some workspace settings that weren't present the last time this file was edited. @calebmshafer should I leave this in, or do another commit that undoes this change?

**Review by bbastings — APPROVED** [review]

**Comment by bbastings** [comment]
> @tcobbs-bentley can you explain in your PR description how this error arises? I'm not sure the ViewLookAndMove tool - or pointer lock in general - makes any sense on a touch device.

The issue is a crash when the tool exits because the value is undefined instead of the expected null. You are correct that pointer lock isn't used and should have no effect for touch input.

**Comment by tcobbs-bentley** [comment]
@pmconne I'll update the description. It is my understanding that ViewLookAndMove was designed to work on touch screens and pop up joysticks under two separate fingers. So it does work on mobile. However, the original report for this problem was someone using a web-based iModel.js app on a mobile device, and using this tool leads to an unhandled exception. So, even if using a web-based iTwin isn't "recommended" on a mobile device, it still shouldn't throw an exception.
